kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/monitor/DhcpServerMonitor.java

```diff
@@ -69,10 +69,14 @@ private void monitor() {
         this.dhcpServerInterfaceConfiguration.entrySet().forEach(entry -> {
             String interfaceName = entry.getKey();
             boolean enable = entry.getValue();
-            if (enable && !this.dhcpServerManager.isRunning(interfaceName)) {
-                startDhcpServer(interfaceName);
-            } else if (!enable && this.dhcpServerManager.isRunning(interfaceName)) {
-                stopDhcpServer(interfaceName);
+            try {
+                if (enable && !this.dhcpServerManager.isRunning(interfaceName)) {
+                    startDhcpServer(interfaceName);
+                } else if (!enable && this.dhcpServerManager.isRunning(interfaceName)) {
+                    stopDhcpServer(interfaceName);
+                }
+            } catch (KuraException e) {
+                logger.warn("Failed to chech DHCP server status for the interface " + interfaceName, e);
```

Comment:
We should log an error here since the monitor is not working.

Reply:
in the other similar condition we use a warning, i replied the behavior.